1001-2000/LIT1295Dersan.xml

```diff
@@ -432,66 +342,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      </ab>
                   </div>
                </div>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamAqabe" corresp="LIT6822SalamLakaAqabe">
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamTahsas" corresp="LIT6822SalamLakaAqabi">
```

Comment:
You kept the reference to the month for this salam, while you changed the IDs for other salams.

Reply:
To save work I refrained from xhanging Id for texts that do not jump around the year